sitexml.js

```diff
@@ -176,9 +198,11 @@ function sitexml (path) {
         return undefined;
     };
 
-    /*
+    /**
      * Returns default theme if PAGE@theme is not defined (see algorithm: http://sitexml.info/algorithms)
-     * */
+	 *
+	 * @return {undefined} theme.
```

Comment:
Type of the return is not `undefined`, it is type of `theme` or `undefined` (returns theme or undefined).